script.js

```diff
@@ -1,4 +1,4 @@
-function GenerateHuman (name, surname, age, phoneNumber, country, id) { 
+function generateHuman (name, surname, age, phoneNumber, country, id) { 
     this.name = name,
     this.surname = surname,
     this.email = `${name}${surname}@test.pl`
```

Comment:
[minor]: normally from input you always receive email in proper format, so you do not need to format it